README.md

```diff
@@ -5,11 +5,1119 @@
+
+```ts
+// Project User model fields `id` & `name` onto User Object
+
+objectType({
+  name: 'User',
+  definition(t) {
+    t.model.name()
+  },
+})
+```
+
+```gql
+type User {
+  name: String
+}
+```
+
+```ts
+// Project User model fields `id` & `name` onto Person Object
+
+objectType({
+  name: 'Person',
+  definition(t) {
+    t.model('User').name()
+  },
+})
+```
+
+```gql
+type Person {
+  name: String
+}
+```
+
+#### `alias` Option
+
+Use `alias` to change the name of the field projected onto the `Object`.
+
+```ts
+objectType({
+  name: 'User',
+  definition(t) {
+    t.model.name({ alias: 'handle' })
+  },
+})
+```
+
+```gql
+type User {
+  handle: String
+}
+```
+
+#### `type` Option
+
+Only available for relational model fields. Use `type` to change the name of the projected field type. This is useful when the model that the field relates to has been projected onto a differently named `Object`.
+
+```ts
+objectType({
+  name: 'Article',
+  definition(t) {
+    t.model('Post').title()
+  },
+})
+
+objectType({
+  name: 'User',
+  definition(t) {
+    t.model.posts({ alias: 'articles', type: 'Article' })
+  },
+})
+```
+
+```gql
+type Article {
+  title: String
+}
+
+type User {
+  articles: [Article]
+}
+```
+
+#### `ordering` `pagination` `filtering` Options
+
+Only available for list type model fields. Please refer to TODO for details.
+
+### `t.crud`
+
+`t.crud` is for exposing operations against your projected models. It is only available inside `Query` and `Mutation` definitions. `t.crud` properties are called "Operation Publishers". They accept configuration allowing you to control how the operation is seen through your GraphQL schema.
+
+`t.crud` reflects a subset of [Photon](https://photonjs.prisma.io)'s capabilities. For every one model in your Prisma schema there are ten corresponding Operation Publishers.
+
+```prisma
+model User {
+  id Int @id
+}
+```
+
+```ts
+queryType({
+  definition(t) {
+    t.crud.user()
+    t.crud.users()
+  },
+})
+
+mutationType({
+  definition(t) {
+    t.crud.createOneUser()
+    t.crud.udpateOneUser()
+    t.crud.upsertOneUser()
+    t.crud.deleteOneUser()
+
+    t.crud.createManyUser()
+    t.crud.updateManyUser()
+    t.crud.upsertManyUser()
+    t.crud.deleteManyUser()
+  },
+})
+```
+
+The following Operation Publisher docs follow this format:
+TODO revise this it is ugly
+
+```
+<overview>
+<contributions to your GraphQL schema>
+<example>
+  <GraphQL client operations>
+  <GraphQL schema>
+  <Nexus/Nexus-Prisma type defs>
+  <Prisma schema model defs>
+```
+
+#### One-Create Operation
+
+Allow clients to create records of the respective model.
+
+Relation fields may be connected with an existing record or a sub-create may be inlined. If the relation is a `List` then multiple connections or sub-creates are permitted.
```

Comment:
> is List an actual programmatic concept in this context? otherwise I suggest lowercase spelling and no code formatting (if it is, is there maybe a link to docs of what that List type is`?

Similar to what I mentioned else where, a GraphQL Type.

> (nested mutations)

👍